core-ui/src/components/Predefined/Details/Pod/ContainerStatus.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { StatusBadge } from 'react-shared';
+
+export function ContainerStatus({ status }) {
+  const { i18n } = useTranslation();
+
+  const state =
+    status.state.running || status.state.waiting || status.state.terminated;
+
+  const message = state.message || null;
+
+  const badgeType = status => {
+    switch (status) {
+      case 'Running':
+      case 'Completed':
+      case 'Succeeded':
+        return 'success';
+      case 'ContainerCreating':
+      case 'Initing':
+      case 'Pending':
+      case 'PodInitializing':
+      case 'Terminating':
+        return 'info';
+      case 'Unknown':
+        return undefined;
+      default:
+        return 'error';
+    }
+  };
+
+  return (
+    <div>
+      <StatusBadge
+        i18n={i18n}
+        resourceKind="containers"
+        additionalContent={message}
+        type={badgeType(state.reason || 'Running')}
+      >
+        {state.reason || 'Running'}
```

Comment:
We don't show tooltips for running/completed statuses.
We have labels like 'imagepullbackoff' even tho we agreed at some point that those should be simplified to just 'error'.

We should use the same approach as we have for pods statuses. Statuses for pods are calculated based on container statuses, so I believe they should display similar messages.